File: dsa-practice/maths/problem6.js
const bmr = (gender,weight,height,age) => {
    let clbmr = 10*weight + 6.25*height + 5*age;
    if(gender ==="male"){
        return clbmr+5
    }
    else{
        return clbmr - 161
    }
}
console.log(bmr("male",47,5.5,20))

File: dsa-practice/maths/problem6.cpp
#include<iostream>
using namespace std;
int main(){
    float weight,height;
    int age;
    float clbmr,bmr;
    char gender[20];
    cout<<"Choose your gender -- male/female: ";
    cin>>gender;
    cout<<"Enter your weight(in kg): ";
    cin>>weight;
    cout<<"Enter your height(in cm): ";
    cin>>height;
    cout<<"Enter your age: ";
    cin>>age;
    clbmr = 10*weight + 6.25*height + 5 * age;
    if(gender=="male"){
        bmr = clbmr +5;
    }
    else{
        bmr = clbmr -161; 
    }
    cout<<endl<<"your bmr is: "<<bmr<<endl;
    return 0;
}